app/models/user.rb

```diff
@@ -1,9 +1,11 @@
 class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please from the project requirement, You should make sure that the app asks for confirmation of email so make sure that you add `:confirmable` and ensure that the user email is confirmed before the signup/login successful :blush: